policy/modules/kernel/corenetwork.te.in

```diff
@@ -241,6 +241,7 @@ network_port(rsync, tcp,873,s0, udp,873,s0)
 network_port(rtsp, tcp,554,s0, udp,554,s0)
 network_port(rwho, udp,513,s0)
 network_port(sap, tcp,9875,s0, udp,9875,s0)
+network_port(securedns, tcp,853,s0)
```

Comment:
I didn't think about this before, but I'm considering if this may make more sense to simply add this to the existing `dns` definition instead.

Reply:
I do not have a strong opinion about this, so I add TCP 853 to the `dns` definition like you suggested.